src/main/java/SeaBattle/Main.java

```diff
@@ -0,0 +1,10 @@
+package SeaBattle;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        StartGame startGame = new StartGame();
```

Comment:
Looks like a fresh view on the course project, fits the requirement, but doesn't look good, not from the interface and not from the code. You show some knowledge in Java Core, I don't see much architecture or right abstractions in the app. Playing is nice though, but x for miss and X for hit looks awful...  

Approved!